### PR TITLE
Feature/expiration date preference

### PR DESCRIPTION
**Pull request by ramiroarguelles**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

**Review by lamaralmp — APPROVED** [review]

File: guides/online-payments/checkout-pro/latest/configurations.en.md [hunk 1]
```diff
@@ -255,6 +255,34 @@ paymentmethods.Installments = 12;
...
+===
+The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
+===
+"date_of_expiration": "2020-05-30T23:59:59.000-04:00";
+```
+]]]
+
+The deadline for approval of the cash payment is between 1 and 2 working days according to the payment method. Therefore, we recommend that you set the due date with at least 3 days to ensure that payment is made.
+
+Check [credit times by payment method](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/Medios-de-pago-y-acreditaci-n_221) when configuring.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.en.md:275]:
```suggestion
Check [credit times by payment method](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/_221) when configuring.
```

File: guides/online-payments/checkout-pro/latest/configurations.es.md [hunk 2]
```diff
@@ -298,6 +298,33 @@ Usa el atributo taxes para definir el valor que corresponda
...
+===
+La fecha usa el formato ISO 8601: yyyy-MM-dd'T'HH:mm:ssz
+===
+"date_of_expiration": "2020-05-30T23:59:59.000-04:00";
+```
+]]]
+
+El período de acreditación es de 1 y 2 días hábiles según el medio de pago. Por lo tanto, te recomendamos establecer la fecha de vencimiento con al menos 3 días para asegurarte de que se realice el pago.
+
+Ten en cuenta los [tiempos de acreditación por medio de pago](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/Medios-de-pago-y-acreditaci-n_221) para realizar la configuración.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.es.md:318]:
```suggestion
Ten en cuenta los [tiempos de acreditación por medio de pago](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/_221) para realizar la configuración.
```

File: guides/online-payments/checkout-pro/latest/configurations.pt.md [hunk 3]
```diff
@@ -277,6 +277,33 @@ _`value`_ | Valor do imposto. Permitido o máximo de duas casas decimais. Para i
...
+===
+A data usa o formato ISO 8601: yyyy-MM-dd'T'HH:mm:ssz
+===
+"date_of_expiration": "2020-05-30T23:59:59.000-04:00";
+```
+]]]
+
+O prazo de creditação está entre 1 e 2 dias úteis de acordo com o meio de pagamento. Por isso recomendamos que você defina a data de expiração com no mínimo 3 dias para garantir que o pagamento seja feito.
+
+Revise os [tempos de creditação por meio de pagamento](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/Medios-de-pago-y-acreditaci-n_221) para executar a configuração.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.pt.md:297]:
```suggestion
Revise os [tempos de creditação por meio de pagamento](https://www.mercadopago[FAKER][URL][DOMAIN]/ajuda/_221) para executar a configuração.
```

File: guides/online-payments/checkout-pro/latest/configurations.es.md [hunk 4]
```diff
@@ -298,6 +298,33 @@ Usa el atributo taxes para definir el valor que corresponda
 
 ------------
 
+----[mla, mlb, mco]----
+
+### Fecha de vencimiento para pagos en efectivo
+
+Por defecto, la fecha de vencimiento para los pagos en efectivo es de 30 días. Si quieres, puedes cambiarla enviando el campo `date_of_expiration` en la solicitud de creación de la preferencia. La fecha configurada debe ser entre 1 y 30 días a partir de la fecha de creación de la preferencia de pago.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.es.md:305]: ¿La fecha de vencimiento por defecto sigue siendo 30 o es 7?

File: guides/online-payments/checkout-pro/latest/configurations.en.md [hunk 5]
```diff
@@ -255,6 +255,34 @@ paymentmethods.Installments = 12;
...
+## Expiration date of cash payment
+
+The default expiration date for cash payments is 30 days. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.
+
+[[[
+```json
+===
+The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
+===
+"date_of_expiration": "2020-05-30T23:59:59.000-04:00";
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.en.md:269]:
```suggestion
"date_of_expiration": "2020-05-30T23:59:59.000-04:00"
```

File: guides/online-payments/checkout-pro/latest/configurations.es.md [hunk 6]
```diff
@@ -298,6 +298,33 @@ Usa el atributo taxes para definir el valor que corresponda
...
+
+[[[
+```json
+===
+La fecha usa el formato ISO 8601: yyyy-MM-dd'T'HH:mm:ssz
+===
+"date_of_expiration": "2020-05-30T23:59:59.000-04:00";
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.es.md:312]:
```suggestion
"date_of_expiration": "2020-05-30T23:59:59.000-04:00"
```

File: guides/online-payments/checkout-pro/latest/configurations.pt.md [hunk 7]
```diff
@@ -277,6 +277,33 @@ _`value`_ | Valor do imposto. Permitido o máximo de duas casas decimais. Para i
...
+## Data de expiração de meios de pagamento em dinheiro
+
+A data de expiração padrão para pagamentos em dinheiro é de 30 dias. Opcionalmente é possível alterar essa data enviando o campo `date_of_expiration` na requisição de criação da preferência. A data configurada deve estar entre 1 e 30 dias a partir da data de emissão.
+
+[[[
+```json
+===
+A data usa o formato ISO 8601: yyyy-MM-dd'T'HH:mm:ssz
+===
+"date_of_expiration": "2020-05-30T23:59:59.000-04:00";
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.pt.md:291]:
```suggestion
"date_of_expiration": "2020-05-30T23:59:59.000-04:00"
```

File: guides/online-payments/checkout-pro/latest/configurations.es.md (hunk 4 above)
Comment (SofiaMoralesGonzalez1992) [inline guides/online-payments/checkout-pro/latest/configurations.es.md:305]:
```suggestion
Por defecto, la fecha de vencimiento puede variar entre 5 a 30 días según el medio de pago en efectivo que estes utilizando. Si quieres, puedes cambiarla enviando el campo `date_of_expiration` en la solicitud de creación de la preferencia. La fecha configurada debe ser entre 5 y 30 días a partir de la fecha de creación de la preferencia de pago.
```

File: guides/online-payments/checkout-pro/latest/configurations.pt.md [hunk 8]
```diff
@@ -277,6 +277,33 @@ _`value`_ | Valor do imposto. Permitido o máximo de duas casas decimais. Para i
 
 ------------
 
+----[mla, mlb, mco]----
+
+## Data de expiração de meios de pagamento em dinheiro
+
+A data de expiração padrão para pagamentos em dinheiro é de 30 dias. Opcionalmente é possível alterar essa data enviando o campo `date_of_expiration` na requisição de criação da preferência. A data configurada deve estar entre 1 e 30 dias a partir da data de emissão.
```
Comment (SofiaMoralesGonzalez1992) [inline guides/online-payments/checkout-pro/latest/configurations.pt.md:284]:
```suggestion
A data de validade pode variar entre 5 a 30 dias dependendo do método de pagamento que você está usando opcionalmente é possível alterar essa data enviando o campo `date_of_expiration` na requisição de criação da preferência. A data configurada deve estar entre 5 e 30 dias a partir da data de emissão.
```

File: guides/online-payments/checkout-pro/latest/configurations.en.md [hunk 9]
```diff
@@ -255,6 +255,34 @@ paymentmethods.Installments = 12;
 ```
 ]]]
 
+----[mla, mlb, mco]----
+
+## Expiration date of cash payment
+
+The default expiration date for cash payments is 30 days. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.
```
Comment (SofiaMoralesGonzalez1992) [inline guides/online-payments/checkout-pro/latest/configurations.en.md:262]:
```suggestion
The default expiration date can vary between 5 to 30 days depending on the means of payment you are using.. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 5 and 30 days from the preference creation date.
```

**Review by SofiaMoralesGonzalez1992 — COMMENTED** [review]
Modifico el texto con la informacion correcta sobre fechas de vencimientos de pagos en efectivo.

File: guides/online-payments/checkout-pro/latest/configurations.en.md (hunk 9 above)
Comment (pamalv7) [reply guides/online-payments/checkout-pro/latest/configurations.en.md:262]:
```suggestion
By default, the due date of a cash payment can vary between 5 and 30 days depending on the payment method you are using. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.
```

File: guides/online-payments/checkout-pro/latest/configurations.es.md (hunk 4 above)
Comment (pamalv7) [reply guides/online-payments/checkout-pro/latest/configurations.es.md:305]:
```suggestion
Por defecto, la fecha de vencimiento de un pago en efectivo puede variar entre 5 y 30 días según el medio de pago estés utilizando. Si quieres, puedes cambiarla enviando el campo `date_of_expiration` en la solicitud de creación de la preferencia. La fecha configurada debe ser entre 1 y 30 días a partir de la fecha de creación de la preferencia de pago.
```

File: guides/online-payments/checkout-pro/latest/configurations.pt.md (hunk 8 above)
Comment (pamalv7) [reply guides/online-payments/checkout-pro/latest/configurations.pt.md:284]:
```suggestion
Por padrão, a data de vencimento para pagamentos em dinheiro pode variar entre 5 e 30 dias, dependendo do meio de pagamento que você está usando. Opcionalmente é possível alterar essa data enviando o campo `date_of_expiration` na requisição de criação da preferência. A data configurada deve estar entre 1 e 30 dias a partir da data de emissão.
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

File: guides/online-payments/checkout-pro/latest/configurations.en.md (hunk 9 above)
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.en.md:262]:
```suggestion
If you want, you can change the default due date of a cash payment by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.
```

File: guides/online-payments/checkout-pro/latest/configurations.pt.md (hunk 8 above)
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.pt.md:284]:
```suggestion
Opcionalmente é possível alterar a data de vencimento por padrão para pagamentos em dinheiro enviando o campo `date_of_expiration` na requisição de criação da preferência. A data configurada deve estar entre 1 e 30 dias a partir da data de emissão.
```

File: guides/online-payments/checkout-pro/latest/configurations.es.md (hunk 4 above)
Comment (pamalv7) [inline guides/online-payments/checkout-pro/latest/configurations.es.md:305]:
```suggestion
Si quieres, puedes cambiarla la fecha de vencimiento por defecto de un pago en efectivo enviando el campo `date_of_expiration` en la solicitud de creación de la preferencia. La fecha configurada debe ser entre 1 y 30 días a partir de la fecha de creación de la preferencia de pago.
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by pamalv7 — APPROVED** [review]

**Review by SofiaMoralesGonzalez1992 — APPROVED** [review]

**Comment by lamaralmp** [comment]
Los cambios serán tomados en el PR #934 por el conflicto de la carpeta `/latest/` que no existe más.

**Review by SofiaMoralesGonzalez1992 — APPROVED** [review]

**Pull request closed without merging**